Apps/WebClient/src/ClientApp/app/views/landing.vue

```diff
@@ -447,5 +532,21 @@ export default class LandingComponent extends Vue {
   private getTileClass(index: number): string {
     return index % 2 == 0 ? "order-md-1" : "order-md-2";
   }
+
+  private get warningIcon(): IconDefinition {
+    return faExclamationTriangle;

```

Comment:
these icons are used on the timeline too. Instead of setting the icons only here we can just load them from the library directly